[CS2113T-F12-1] CardLI

xRossKoh opened the pull request:
CardLI is a desktop app for creating, organising, and reviewing flashcards via a Command Line Interface to serve students.

renzocanare commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+* Saves all the decks
+* Saves all the flashcards
+* Remembers which deck each flashcard belongs to
+* Saves the results of each test
```
You can consider adding a sequence diagram here to show how the data is loaded into the app and saved into the text files. This will help your users to understand more about how the data is safely saved, and also explain your earlier point in the Design section on "initializes the components in the correct sequence". 

![image](https://user-images.githubusercontent.com/69526313/138994311-3b33890e-a785-49e2-aa54-f7dd1e7d33ef.png)

renzocanare commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+</body>
+
+
+`readCardsFromFile()` and `readTestsFromFile()`
```
This section appears to have been formatted incorrectly and might have caused the rest of the DG to break. The image below is what this section looks like at your GitHub Pages DG link.

![image](https://user-images.githubusercontent.com/69526313/138995282-f00c3111-7c70-4e40-802b-cbe6d95f18f9.png)

renzocanare commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+will be printed one at a time for the user to answer. The user's answer is then parsed and then added into
+the `AnswerList`. This process is repeated for the entire `Deck` that is being tested.
+
+![sequence diagram](../docs/assets/markTestSequenceDiagram.png)
```
Great choice to use sequence diagrams here, but it seems that the sequence diagrams are not appearing correctly in the GitHub Pages for your DG (as seen below).

You may want to consider removing `../docs/` from the image source in the code (e.g. `assets/testClassDiagram.png` as the image link can be relative to where `DeveloperGuide.md` is stored.

![image](https://user-images.githubusercontent.com/69526313/138996028-ad57b0f8-6fab-4449-bab2-3e1853d89ed0.png)

renzocanare commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+
+Given below is the sequence diagram for `find`:
+
+![](assets/findFlashcardDiagram.png)
```
Good choice to add single-level numbering as it helps me to understand the sequence of which the calls flow. However, I personally am unsure if this is standard notation, and you may want to check if this is allowed. If it is, it's great help! 

However, the image also appears quite small on your DG Page, and requires a few zoom-ins to read. If this is possible to adjust, it will definitely help the overall readability of your DG!

![image](https://user-images.githubusercontent.com/69526313/138997033-48c868b2-4af7-4d44-a482-e1ff0e0b7b34.png)

renzocanare commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+
+`returnMatchingFlashCards()` is implemented by creating a stream that consists of all the `FlashCards` from one deck, and filters them based on whether they contain the search term given. Finally all the `FlashCards` that contain the search term are collected in an arrayList and their console outputs are returned in string format for `CardLiUi` to display to the user.
+
+Given below is the sequence diagram for `find`:
```
You may want to consider moving your sequence diagram for `find` to below the header as that is where many will expect to find images related to a whole section (and to keep in line with the formatting of the rest of your DG, where their sequence diagrams are located below the header of that section).

renzocanare reviewed (commented):
Overall, great work on the Developer Guide and use of sequence diagrams to supplement your wordier sections. With a little bit of quick-fixes on the formatting, the whole Developer Guide will look fantastic!

aditichadha1310 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+
+### `EditCardCommand`
+
+![](assets/editCardCommandSeqDiagram.png)
```
![image](https://user-images.githubusercontent.com/70324480/138998166-52e96a67-6b55-4d0c-80ef-b8d94263a474.png)
You might want to eliminate the lifeline of the activated bars once they are no longer being referenced. 
I mean that once there is a cross in the line, which indicates the item is no longer being referenced you can choose to not extend the line after that cross.

aditichadha1310 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+
+### Move
+
+![](assets/moveCardCommandSeqDiagram.png)
```
![image](https://user-images.githubusercontent.com/70324480/138998400-7aa1f6cd-453b-4d04-a89d-3cf2a6f6f407.png)
Similarly here, your team might want to not extend the timeline after there is a cross (which represents that the object is no longer being referenced).

aditichadha1310 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+
+### Test Feature
+
+![class diagram](../docs/assets/testClassDiagram.png)
```
![image](https://user-images.githubusercontent.com/70324480/138999149-6e65478a-2948-4f64-b8f3-0956520fcf6f.png)
It seems that the image here is missing. You all might want to check this and update the DG.

aditichadha1310 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+
+How the parsing works:
+
+`Inner Parser` or `Outer Parser` creates a `XYZCommand` which in turn creates its corresponding `XYZCommandParser` (eg. when `InnerParser` creates a `EditCardCommand`, `EditCardCommand` creates a `EditCardParser`.)</li>
```
![image](https://user-images.githubusercontent.com/70324480/138999373-dae7aa8c-b387-4c0d-a6c2-f603f5f97a4f.png)
Your team might consider checking the content presented on the DG. For example, here the DG displays a List tag  which is not supposed to be displayed. 
Otherwise, its a good job done on the content:)

aditichadha1310 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+3. The command communicates with the `Model` when it is executed (eg. to add a flashcard).
+4. The result of the command execution is encapsulated as a `CommandResult` object which is returned to the `UI` from `Logic` to process.
+
+![](assets/parserArchitectureDiagram.png)
```
![image](https://user-images.githubusercontent.com/70324480/138999763-c0d9272d-4f3e-4f8e-aa13-9a45911c3f9b.png)

Your team might consider making the direction of these little triangle arrows more explicit for better understandability and redability.

aditichadha1310 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,327 @@
+| |user| add flashcards|
+| |user| delete flashcards|
+| |user| view my flashcards|know what cards I currenly have in the deck|
+| |user| test myself with my flashcards|know if I have memorised the flashcards correctly|
```
![image](https://user-images.githubusercontent.com/70324480/139000328-23a3ff1a-346c-48dc-b6a9-1cfec1155aa1.png)
As from the syntax used, I think in this section your team was trying to put forward the information in tabular form. However there are some formatting issues which do not present in the tabular form. I have faced a similar issue before and I think its because too much information is being presented in one line. 
You can consider making the points (which are presented on each line) to be short so that it fits well in the table and can be presented in tabular form.

MAZJ124 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
 # Developer Guide
```
Currently on Github pages, the template used for this developer guide only takes up half of the screen (at least on my device), perhaps it is good to change to a template that is easier for the reader to view?
![image](https://user-images.githubusercontent.com/69350115/139169175-f395c8d6-888b-4ea9-9e16-237a120b97a5.png)

MAZJ124 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the
+original source as well}
+
```
Perhaps your team can add something like a content page here containing hyperlinks to jump to each section of the DG? This might make navigating around the document easier.

MAZJ124 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+
+### Model Component
+
+![](assets/modelArchitectureDiagram.png)
```
![image](https://user-images.githubusercontent.com/69350115/139169615-a2a62a17-6d27-440f-8be2-d87a46d549ec.png)
Maybe the dotted line with these arrows can be straight instead (especially when there is enough space) since the direction of the arrow might seem a little bit confusing to the reader.

MAZJ124 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+### Move
 
-## Design & implementation
+![](assets/moveCardCommandSeqDiagram.png)
```
![image](https://user-images.githubusercontent.com/69350115/139170617-953088fa-4835-4c57-b24e-51c0b96aaa81.png)
Having a sequence diagram here really helps to clear things up, however, unlike the previous diagrams, this one (and the next few) tends to have more content inside, and it is kind of hard for the reader to read, perhaps the size of the image can be enlarged?
On top of that, I think it was also mentioned by someone else as well: I believe that the lifeline of an object do not need to be drawn after the cross.

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
 # Developer Guide
 
+# Introduction
```
Would it be better for users to navigate around if you consider creating a 
"table of contents" section with hooks (a.k.a. hyperlinks)?

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+
+Given below is the sequence diagram for `edit` (Deck):
+
+![](assets/editDeckCommandSeqDiagram.png)
```
You could perhaps consider using create in your sequence diagrams to indicate newly created objects.
You could consider this for DeckManager, Deck, or CardLiUi
![image](https://user-images.githubusercontent.com/69447277/139174229-3f727fb7-37d8-4eab-beed-99dbdfcd4487.png)

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+### `EditDeckCommand`
+
+
+Given below is the sequence diagram for `edit` (Deck):
```
You could perhaps consider deleting the lifeline upon object deletion
![image](https://user-images.githubusercontent.com/69447277/139174175-83d3f90d-a41f-4db3-a764-bd551be127b5.png)

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+
+Given below is the sequence diagram for `find`:
+
+![](assets/findFlashcardDiagram.png)
```
More of a consistency issue, perhaps you could adjust the sequence diagrams for move, find, and editcardcommand to follow the same style. For instance, in object deletion, if you want to use the cross, consider using it for all your diagrams. 

For the find sequence diagram in particular, well done in using the numbering, perhaps this could be replicated in the other diagrams as well. Also, "create" is used here, which again could be replicated in the other sequence diagrams if applicable.

You could also take an extra step by adding numbered steps in the explanation and adjusting it according to the numbers in the sequence diagram

![image](https://user-images.githubusercontent.com/69447277/139174873-1f9813a4-1d5e-413c-8dfc-c7b7dc9d7ce9.png)

LouisLouis19 replied:
You could perhaps consider adding the operation invoked above the arrows in steps 2 and 3
![image](https://user-images.githubusercontent.com/69447277/139176593-e8d1ff8c-8d69-40a9-bd10-fff82b90bb30.png)

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+returns `deck`, of string type, which represents the deck to be edited.`prepareMoveCommand()` will then return
+a string array, `preparedArguments`, which represents the arguments for the next method call.
+
+The `execute()` method will then call the `moveCard()` method of the `DeckManager` class, which in turn calls the
```
The sequence diagram explanation is detailed, to enhance it further, you could consider using bullet points to ease readers in understanding the thought process better.

EdisonZhong17 replied:
In this part can consider to use ```FUNCTION_NAME``` to make the long paragram more clearer.

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+prompt the user to input a number corresponding to the index of the deck that they want to 
+be tested on or "all" to test all decks.
+
+![sequence diagram](../docs/assets/getTestDeckSequenceDiagram.png)
```
Unfortunately, the class and sequence diagrams for the Test feature are not rendered properly. Could try to fix this urgently

![image](https://user-images.githubusercontent.com/69447277/139175183-b475490d-4f8a-4d1f-88bf-49f34b3076ad.png)

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+namely an `ArrayList` of a generic type `<T>` as well as a `boolean` value, `saveCards`, that indicates whether the method
+is saving cards or tests to their respective text files.
+
+![](assets/writeToFileSequenceDiagram.png)
```
Similar to the previous sequence diagrams, perhaps you could consider making your diagram styling more consistent.

LouisLouis19 replied:
You could perhaps consider using "create" for newly created objects, ie. FileWriter, ArrayList

![image](https://user-images.githubusercontent.com/69447277/139175645-04ab9783-06ee-4920-99a3-0f40c4598b96.png)

LouisLouis19 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+* Has a lot of flashcards
+
 
 ### Value proposition
```
Well done in keeping this section concise. However, a bug caused the page to not render properly.
The user stories, non-functional requirements, glossary, and instructions for manual testing sections are not rendered as expected.

![image](https://user-images.githubusercontent.com/69447277/139175829-d57ddab8-75d3-4e68-876f-f57dbd5870ef.png)

EdisonZhong17 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+CardLI is a Command Line Interface (CLI) desktop app designed to help students manage their flashcards. CardLI can help
+students keep track of all their flashcards. It also allows students to test their knowledge. All of this in one
+single platform.
+
```
Before the start of DG, it would be better to have a set of hyperlinks for user to navigate the whole DG.

EdisonZhong17 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+
+### Test Feature
+
+![class diagram](../docs/assets/testClassDiagram.png)
```
![image](https://user-images.githubusercontent.com/69306115/139372368-d66d42a3-057b-4ca7-952d-af0171f6fb36.png)
some of the diagrams are not showing properly, maybe there's something wrong when you imported this .png file to your DG.

EdisonZhong17 commented on docs/DeveloperGuide.md:
```diff
@@ -1,29 +1,366 @@
+the method. 
+
+![](assets/readTestsFromFileSequenceDiagram.png)
+
```
![image](https://user-images.githubusercontent.com/69306115/139372700-c5a374c0-5abd-4180-94f5-86240145538b.png)
For this sequence diagram, maybe you can consider adding cross after each activation bar is ended.

xkisxk committed:
Fixed bug where commands with no input accepts input

xkisxk committed:
Fixed bug

xRossKoh committed:
implemented 2dp printing for test result but could not test due to timer not working properly in intellij

xRossKoh committed:
solved checkstyle issues

Commit:
Edit find function to return error message

Edited find function to return error message when there are no cards matching the search term.

Commit:
Handle large indexes, delete the delete by input function

xkisxk committed:
Added more testing

Commit:
Fixed more bugs in delete card and edit card

ThaddeusLim99 committed:
Fixed UG bug and EditDeck bug

Commit:
Handled very big integer strings with number format exception

ThaddeusLim99 committed:
fixed checkStyletest

ThaddeusLim99 committed:
fiexed EditCard bugs

ThaddeusLim99 committed:
fixed out of bounds error for EditCardCommand

Commit:
change split parameters

changed split parameters such that there needs to be a space after a flag

JWweiyin committed:
Merge pull request #166 from xkisxk/fix-bugs

Fixed dg bugs and bugs from PED

JWweiyin committed:
Merge branch 'master' into master

JWweiyin committed:
Merge pull request #168 from astralum/master

Fix output when viewing a flashcard to accommodate long lines

JWweiyin committed:
Merge branch 'master' into json

JWweiyin committed:
Merge pull request #171 from xRossKoh/json

Changed storage and test functionalities

JWweiyin committed:
Merge branch 'master' into fix-bugs

JWweiyin committed:
Merge pull request #172 from JWweiyin/fix-bugs

Bug fixes for v2.1

JWweiyin committed:
Merge branch 'master' into week-11

JWweiyin committed:
Merge pull request #173 from ThaddeusLim99/week-11

Fixed bugs for EditDeck,EditCard,UG; Added Junit Testing for EditDeck,EditCard

Commit:
fix commands that dont take other arguments

ThaddeusLim99 committed:
Merge branch 'week-11'

xkisxk committed:
Added Junit testing

xkisxk committed:
Merge branch 'master' into fix-bugs

xkisxk committed:
Added junit testing for TestHistory

xkisxk committed:
Moved testing and reviewing to be called inside the command

xkisxk committed:
fix check error

ThaddeusLim99 committed:
Fixed move and edit command, did testing for move and my PPP

ThaddeusLim99 committed:
added logo to README

ThaddeusLim99 committed:
deleted incorrect test

ThaddeusLim99 committed:
changed expected.txt to pass check

ThaddeusLim99 committed:
changed file name

xkisxk committed:
Updated DG to current implementation of test, added PPP

xRossKoh committed:
Updated personal PPP

xkisxk committed:
Reorganized ppp

Commit:
Add more screenshots to ug, refine ug and help commands

refined ug and help commands to align with our current implementation

astralum committed:
Merge pull request #175 from xkisxk/fix-bugs

Added Junit testing, moved testing/reviewing into be executed inside the respective command.

astralum committed:
Merge branch 'master' into week-12

astralum committed:
Merge pull request #176 from ThaddeusLim99/week-12

Fixed move and edit;did testing for move;did PPP draft;added logo to UGDG

astralum committed:
Merge pull request #177 from xRossKoh/master

Updated personal PPP

astralum committed:
Merge branch 'master' into master

astralum committed:
Merge pull request #179 from JWweiyin/master

Refined UG and Help commands

astralum committed:
Change timer so that it runs for the entire duration of the test

astralum committed:
Fix checkstyle violations

astralum committed:
Add Javadoc to Countdown

astralum committed:
Remove unused variables from Countdown

astralum committed:
Make timer update more frequently, add clearScreen() method

astralum committed:
Make answers submitted after timer runs out ungraded

astralum committed:
Fix checkstyle violations

astralum committed:
Restore normal timer duration

astralum committed:
Fix cosmetic error when viewing flashcard

Commit:
PPP done

Commit:
WY PPP done

JWweiyin committed:
Merge pull request #181 from JWweiyin/master

WY PPP done

astralum committed:
Create skeleton for PPP

astralum committed:
Add sections for Features and Enhancements Implemented

astralum committed:
Merge pull request #182 from astralum/branch-ppp

Branch PPP

Commit:
PPP refined

ThaddeusLim99 committed:
updated PPP

ThaddeusLim99 committed:
edited PPP

ThaddeusLim99 committed:
added comments

JWweiyin committed:
Merge pull request #184 from ThaddeusLim99/week-12

updated PPP

JWweiyin committed:
Merge pull request #183 from JWweiyin/master

PPP refined, edited some diagrams in DG

JWweiyin committed:
Merge pull request #180 from astralum/branch-timer

Make timer run for the entire duration of test and display correctly

Commit:
Authoring, cleaning up

JWweiyin committed:
Merge pull request #185 from JWweiyin/master

Authoring, cleaning up

xRossKoh committed:
Set theme jekyll-theme-midnight

xRossKoh committed:
Set theme jekyll-theme-hacker

xRossKoh committed:
Set theme jekyll-theme-slate

Commit:
Reordered user guide

JWweiyin committed:
Merge pull request #186 from JWweiyin/master

Reordered user guide

Commit:
Add function to check whether any other cards have the same front when adding card or editing card.

Commit:
Add command to check if other cards in the app have the same name for add and edit card commands

astralum committed:
Remove unnecessary condition checking, change empty answer in test

xkisxk committed:
Updated test uml diagrams

xkisxk committed:
Added a purpose section and anchor links

xRossKoh committed:
Added sequence diagrams to uml folder for storage class methods

xRossKoh committed:
Fixed minor inaccuracies with certain seq diagrams

xRossKoh committed:
Added ref boxes to parseDeck and parseAnswerList seq diagrams

xkisxk committed:
Merge branch 'master' of https://github.com/xkisxk/tp into updates

xkisxk committed:
testing nested lists

ThaddeusLim99 committed:
edited DGUG

ThaddeusLim99 committed:
added some anchor links

xkisxk committed:
test ug

xkisxk committed:
test ug

ThaddeusLim99 committed:
updated content table

astralum committed:
Add acknowledgements in DG

ThaddeusLim99 committed:
Set theme jekyll-theme-slate

xkisxk committed:
test ug

xkisxk committed:
Added anchor links

xkisxk committed:
anchor link not working

xkisxk committed:
Fixed anchor links

ThaddeusLim99 committed:
finished toc for DG

ThaddeusLim99 committed:
Merge branch 'week-12' of https://github.com/ThaddeusLim99/tp into week-12

ThaddeusLim99 committed:
fixed user stories formatting in DG

xRossKoh committed:
Rewrote DG for storage and added in the necessary sequence diagrams

ThaddeusLim99 committed:
hopefully fix DG anchorlink bug

xRossKoh committed:
Changed the wording of FAQs regarding storage in UG

ThaddeusLim99 committed:
testing

xkisxk committed:
Edited how to

ThaddeusLim99 committed:
fix DG

ThaddeusLim99 committed:
fixed DG again

ThaddeusLim99 committed:
test

ThaddeusLim99 committed:
return

xRossKoh committed:
Added diagram for Cards_CardLI.json

xRossKoh committed:
Changed the way exceptions are handled for readCardsFromFile() and readTestsFromFile() methods

xRossKoh committed:
changed the way exceptions are handled for readCardsFromFile and readTestsFromFile methods within Storage class

xRossKoh committed:
Edited EXPECTED.TXT

ThaddeusLim99 committed:
Added javadoc and updated Seq diagrams

ThaddeusLim99 committed:
fixed comments

ThaddeusLim99 committed:
added glossary

xkisxk committed:
Edited dg, ug

ThaddeusLim99 committed:
edited javadoc

ThaddeusLim99 committed:
Updated PPP

xkisxk committed:
Edited ppp

Commit:
Wrote tests for edit card command

xRossKoh committed:
Added DeckTest and toString and toJSONObject tests in FlashCardTest and AnswerTest

xRossKoh committed:
Removed delete card and edit card tests from DeckTest

xRossKoh committed:
Added toString and toJsonObect Test in AnswerListTest

Added authorship to implemented tests

xRossKoh committed:
Added JavaDoc to Storage class and toJsonObject methods

xRossKoh committed:
Fixed certain checkstyle errors

Commit:
add tests for AddCardCommand and AddDeckCommand

Commit:
no message

Commit:
edited findSeqDiagram

JWweiyin committed:
Merge pull request #187 from JWweiyin/master

Function to check if other cards have the same front

JWweiyin committed:
Merge pull request #188 from xkisxk/updates

Updated ug and fixed check errors

JWweiyin committed:
Merge branch 'master' into master

JWweiyin committed:
Merge pull request #189 from astralum/master

Remove unnecessary condition checking, change empty answer in test, add DG acknowledgements

JWweiyin committed:
Merge pull request #190 from xRossKoh/final_touches

UGDG final touches

JWweiyin committed:
Merge branch 'master' into week-12

JWweiyin committed:
Merge pull request #191 from ThaddeusLim99/week-12

Added TOC and anchor links to DG; Added Javadoc

Commit:
Edit DG, UG, fix checkstyle errors

JWweiyin committed:
Merge pull request #192 from JWweiyin/master

Edit DG, UG, fix checkstyle errors

astralum committed:
Suppress warnings in Storage

astralum committed:
Add Javadoc for InnerParser and OuterParser, edit Javadoc for Countdown

astralum committed:
Add Javadoc for Deck, DeckManager and InnerParser, remove some newlines

astralum committed:
Add Javadoc for Command and CommandResult, remove some newlines from Answer

astralum committed:
Add Javadoc for class descriptors for all Command classes in deck

astralum committed:
Add period to fix checkstyle violation

astralum committed:
Add Countdown to DG

Commit:
DG manual testing: Added test cases for adding deck, entering deck, finding flashcards, adding flashcards, deleting flashcards

xkisxk committed:
Updated dg

ThaddeusLim99 committed:
edited DG

ThaddeusLim99 committed:
Merge branch 'week-12' of https://github.com/ThaddeusLim99/tp into week-12

ThaddeusLim99 committed:
Merge branch 'week-12' into master

ThaddeusLim99 committed:
Merge pull request #3 from AY2122S1-CS2113T-F12-1/master

merge conflict

astralum committed:
Update UML sequence diagram for Countdown

xkisxk committed:
Fixed sequence diagrams

xkisxk committed:
Added manual testing

ThaddeusLim99 committed:
added  some manual testing

astralum committed:
Edit Countdown portion of DG to be more accurate

xRossKoh committed:
Added logging to Storage class and removed certain error messages

ThaddeusLim99 committed:
added manual testing

xRossKoh committed:
Added diagram for Tests_CardLI.json in DG

xRossKoh committed:
Renamed json file screenshots and changed DG accordingly

xRossKoh committed:
Merge pull request #197 from xRossKoh/master

Added logging to Storage class and removed certain error messages

ThaddeusLim99 committed:
added pic

ThaddeusLim99 committed:
resized pic

ThaddeusLim99 committed:
edited pic

ThaddeusLim99 committed:
updated link

ThaddeusLim99 committed:
fixed

JWweiyin committed:
Merge pull request #194 from astralum/branch-dg

Add SuppressWarning, add Javadoc, add Countdown to DG

Commit:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-1/tp

# Conflicts:
#	docs/DeveloperGuide.md
#	docs/assets/dg diagrams/Tests_CardLI.txt Example.png

JWweiyin committed:
Merge pull request #195 from JWweiyin/master

DG manual testing

Commit:
Merge branch 'dg-updates' of https://github.com/xkisxk/tp into xkisxk-dg-updates

# Conflicts:
#	docs/DeveloperGuide.md

Commit:
Merge branch 'xkisxk-dg-updates'
# Please enter a commit message to explain why this merge is necessary,
# especially if it merges an updated upstream into a topic branch.
#
# Lines starting with '#' will be ignored, and an empty message aborts
# the commit.

JWweiyin committed:
Merge pull request #200 from JWweiyin/master

merge

JWweiyin committed:
Merge branch 'master' into week-12

JWweiyin committed:
Merge pull request #199 from ThaddeusLim99/week-12

DG Manual Testing

Commit:
Changed formattting of titles

JWweiyin committed:
Merge pull request #201 from JWweiyin/master

Changed formattting of titles

Commit:
Added json screenshots to folder

JWweiyin committed:
Merge pull request #202 from JWweiyin/master

Added json screenshots to folder

Commit:
JSON screenshots added to my sections of the manual testing

ThaddeusLim99 committed:
changed manual testing format

ThaddeusLim99 committed:
Merge branch 'AY2122S1-CS2113T-F12-1:master' into week-12

Commit:
fixed alignment of screenshots

Commit:
checkstyle tests now ok

ThaddeusLim99 committed:
fixed manual testing

JWweiyin committed:
Merge pull request #203 from JWweiyin/master

i just reformatted some stuff

JWweiyin committed:
Merge pull request #204 from ThaddeusLim99/week-12

fixed manual testing

xRossKoh committed:
Manual instructions edits

xRossKoh committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-1/tp

xRossKoh committed:
added deleting a deck section in manual testing of DG

Commit:
changed error commands

JWweiyin committed:
Merge pull request #205 from JWweiyin/master

changed error message

xRossKoh committed:
minor changes to deleting a deck section in manual instructions of dg

Commit:
fixed aboutme format

ThaddeusLim99 committed:
removed pic

JWweiyin committed:
Merge pull request #206 from xRossKoh/master

Added deleting a deck section under manual instructions in dg

Commit:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-1/tp

xkisxk committed:
Changed formatting of manual testing

ThaddeusLim99 committed:
Merge branch 'AY2122S1-CS2113T-F12-1:master' into week-12

ThaddeusLim99 committed:
added sections 9.1 and 9.2 to DG

xkisxk committed:
edited ppp and dg

astralum committed:
Add Testing section to DG Manual Testing

astralum committed:
Add Reviewing section in DG Manual Testing

astralum committed:
Fix typo in UG

astralum committed:
Add timer explanation to Test flashcards in UG

xRossKoh committed:
Changed deleting a deck section under manual testing instructions in DG to be written in prose

xRossKoh committed:
Added instructions on how to view JSON file

ThaddeusLim99 committed:
update PPP

xRossKoh committed:
Edited AboutUs.md, PPP

xRossKoh committed:
added final touches to PPP

ThaddeusLim99 committed:
Added testing on system level

ThaddeusLim99 committed:
edited name of test functions

astralum committed:
Merge pull request #207 from xkisxk/update-dg

Update dg

astralum committed:
Merge pull request #208 from astralum/branch-dg

Add testing and reviewing to DG, add explanation of timer to UG

astralum committed:
Merge pull request #209 from xRossKoh/master

Final touches

astralum committed:
Merge branch 'master' into week-12

astralum committed:
Merge pull request #210 from ThaddeusLim99/week-12

Some testing and some DG and some PPP

JWweiyin committed:
formatting in manual testing

astralum committed:
Add PPP

JWweiyin committed:
about us formatting

JWweiyin committed:
Update AboutUs.md

xkisxk committed:
final updates to dg

astralum committed:
Merge pull request #212 from astralum/master

Add PPP

xRossKoh committed:
Update xrosskoh.md

xkisxk committed:
Merge pull request #213 from xkisxk/update-dg

final updates to dg

xRossKoh committed:
Update xrosskoh.md

JWweiyin committed:
Update jwweiyin.md

ThaddeusLim99 committed:
Update thaddeuslim99.md

Commit:
made uml diagrams smaller

JWweiyin committed:
Merge branch 'master' into master

JWweiyin committed:
Merge pull request #214 from JWweiyin/master

made diagrams smaller

astralum committed:
Change colour of class in UML sequence diagram

astralum committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-1/tp

astralum committed:
Merge branch 'AY2122S1-CS2113T-F12-1:master' into master

astralum committed:
Merge pull request #215 from astralum/master

Edit Countdown sequence diagram

xkisxk committed:
fixed dg

xkisxk committed:
Merge pull request #216 from xkisxk/update-dg

fixed dg

JWweiyin committed:
Update UserGuide.md

astralum committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-1/tp

xkisxk committed:
Fixed dg

xkisxk committed:
Merge pull request #217 from xkisxk/update-dg

Fixed dg

astralum committed:
Fix bug when adding command

astralum committed:
Merge branch 'master' of https://github.com/AY2122S1-CS2113T-F12-1/tp

xkisxk committed:
Merge pull request #218 from astralum/master

Fix bug when adding command

astralum committed:
Add Javadoc for CommandArgumentsParser class

xkisxk committed:
added javadocs

xkisxk committed:
Merge pull request #220 from xkisxk/master

added javadocs

xkisxk committed:
Merge pull request #219 from astralum/master

Add Javadoc for CommandArgumentsParser class

xkisxk committed:
fix check erros

xkisxk committed:
Merge branch 'master' of https://github.com/xkisxk/tp

xkisxk committed:
Merge pull request #221 from xkisxk/master

w

xkisxk committed:
sorted out authorship in ugdg. updated ppp

xkisxk committed:
Merge pull request #222 from xkisxk/master

sorted out authorship in ugdg. updated ppp
